README.md

```diff
@@ -1 +1,143 @@
+</p>
+
+<p align="center">
+  <a href="https://codecov.io/gh/liftedinit/manifest-ledger">
+    <img src="https://codecov.io/gh/liftedinit/manifest-ledger/branch/reece/codecov/graph/badge.svg" alt="codecov"/>
+  </a>
+  <a href="https://goreportcard.com/report/github.com/liftedinit/manifest-ledger">
+    <img src="https://goreportcard.com/badge/github.com/liftedinit/manifest-ledger" alt="Go Report Card"/>
+  </a>
+  <a href="https://discord.gg/kQkaJzxvk9">
+    <img src="https://badgen.net/badge/icon/discord?icon=discord&label" alt="Discord"/>
+  </a>
+</p>
+
+## Overview
+
+The Manifest Network, built on the Cosmos SDK, is a blockchain tailored for decentralized AI infrastructure access. Initially employing a Proof of Authority (PoA) model it ensures a secure and efficient launch phase, with a trusted validator set managing consensus.
+
+While PoA offers immediate stability and control, the Manifest Network aspires for greater decentralization. The future roadmap includes evolving towards a Proof of Stake (PoS) network, utilizing the underlying CometBft consensus mechanism inherent in the Cosmos SDK.
+
+## Table of Contents
+
+- [System Requirements](#system-requirements)
+- [Installation](#install--run)
+- [Testing](#testing)
+- [Helper](#helper)
+- [Modules](./MODULE.md)
+- [Validators](./network/manifest-1/POST_GENESIS.md)
+- [Multi Sig Guide](./network/manifest-1/MULTI_SIG.md)
+- [Contributing](./CONTRIBUTING.md)
+- [Security/Bug Reporting](./SECURITY.md)
+
+## System Requirements
+
+**Minimal**
+
+- 4 GB RAM
+- 100 GB SSD
+- 3.2 x4 GHz CPU
+
+**Recommended**
+
+- 8 GB RAM
+- 100 GB NVME SSD
+- 4.2 GHz x6 CPU
+
+**Software Dependencies**
+
+1. The Go programming language - <https://go.dev/>
+2. Git distributed version control - <https://git-scm.com/>
+3. Docker - <https://www.docker.com/get-started/>
+4. GNU Make - <https://www.gnu.org/software/make/>
+
+**Operating System**
+
+- Linux (x86_64) or Linux (amd64) Recommended Arch Linux
+
+**Arch Linux:**
+
+```
+pacman -S go git gcc make
+```
+
+**Ubuntu Linux:**
+
+```
+sudo snap install go --classic
+sudo apt-get install git gcc make jq
+```
+
+## Install & Run
+
+Clone the repository from GitHub and enter the directory:
+
+```bash
+    git clone https://github.com/liftedinit/manifest-ledger.git
+    cd manifest-ledger
+```
+
+Then run:
+
+```bash
+    make install
+    mv $GOPATH/bin/manifestd /usr/local/bin
+    manifestd
+```
+
+## Testing
+
+There are various make commands to run tests for the modules with custom implementations
+
+**To test the Proof of Authority implementation run:**
+
+```bash
+make ictest-poa
+```
+
+**To test the Token Factory implementation run:**
+
+```bash
+make ictest-tokenfactory
+```
+
+**To test the Manifest module which includes inflation changes run:**
+
+```bash
+make ictest-manifest
+```
+
+**To test the IBC implementation run:**
+
+```bash
+make ictest-ibc
```

Comment:
Doesn't work as-is.

```
cd interchaintest && go test -race -v -run TestPOA . -count=1
go: downloading golang.org/x/net v0.21.0
=== RUN   TestPOA
    logger.go:130: 2024-03-18T10:05:56.280-0400 INFO    Using embedded configured chains
    logger.go:130: 2024-03-18T10:05:57.225-0400 ERROR   Failed to pull image    {"error": "Error response from daemon: pull access denied for manifest, repository does not exist or may require 'docker login': denied: requested access to the resource is denied", "repository": "manifest", "tag": "local"}
    interchain_builder.go:114: 
                Error Trace:    /home/fmorency/go1.22.1/bin/pkg/mod/github.com/strangelove-ventures/interchaintest/v8@v8.1.0/interchain_builder.go:114
                                                        /home/fmorency/go1.22.1/bin/pkg/mod/github.com/strangelove-ventures/interchaintest/v8@v8.1.0/interchain_builder.go:130
                                                        /home/fmorency/dev/manifest-ledger/interchaintest/poa_test.go:37
                Error:          Received unexpected error:
                                failed to create faucet accounts: failed to create common account with name faucet: failed to create key with name "faucet" on chain poa: image manifest:local: pull image manifest:local: Error response from daemon: pull access denied for manifest, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
                Test:           TestPOA
    setup.go:210: Pruned 2 volumes, reclaiming approximately 0.0 MB
    setup.go:242: Pruned unused networks: [interchaintest-elsgjhba]
--- FAIL: TestPOA (2.26s)
FAIL
FAIL    github.com/liftedinit/manifest-ledger/interchaintest    2.756s
FAIL
make: *** [Makefile:108: ictest-poa] Error 1
```

Reply:
note: need to add `make local-image` to this example before running e2e test

Reply:
added make local-image to the install commands, make ictest-* will work now